pennylane/kernels/__init__.py

```diff
@@ -0,0 +1,33 @@
+# Copyright
```

Comment:
```suggestion
# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Reply:
This is the header in all other PL files...but I just realised that a new file should probably start the copyright range with its creation year? @co9olguy, do you know?

Reply:
I noticed that there are test files that do not have the last year updated to 2021 yet.

Reply:
I think we update every file that we change (unless someone forgets :). But should implement this consistently, good point!

Reply:
yes the copyright should be for the entire repository, not individual source files (exceptions exist if we are leveraging material from elsewhere which already has a license). It's good practice to have them all agree (and we try to update them at the start of the year), but sometimes people forget when they update files

Reply:
so even for files created in 2021, we should practice putting "Copyright 2018-2021"